app/jobs/send_daily_triage_email_job.rb

```diff
@@ -28,4 +28,14 @@ def skip_daily_email?(user)
   def email_decider(user)
     @email_decider ||= EmailDecider.new(user.days_since_last_clicked, minimum_frequency: user.email_frequency)
   end
+
+  DEFAULT_EMAIL_TIME_OF_DAY = Time.utc(2000, 1, 1, 17, 00, 0)
+
+  def before_email_time_of_day?(user)
```

Comment:
Comparing times as strings seems weird. Why not just compare the times directly?